samples/boards/espressif/ulp/lp_core/debug_ulp/README.rst

```diff
@@ -0,0 +1,123 @@
+.. zephyr:code-sample:: debug-ulp
+   :name: Debug ULP
+
+   Sample to demonstrate how to debug the LP Core in ESP32C6.
```

Comment:
sorry for not catching earlier, as per sample guidelines "It is recommended to word this as if you were completing
   the sentence "This code sample shows how to ..."" so:

```suggestion
   Debug the LP Core on ESP32C6.
```

virtual +1 otherwise :) this is very cool stuff!